plugin/src/main/scala/Plugin.scala

```diff
@@ -10,10 +10,16 @@ object DatatypePlugin extends AutoPlugin {
 
   object autoImport {
     val generateDatatypes = taskKey[Seq[File]]("Generate datatypes.")
+    val createDatatypes = settingKey[Boolean]("Whether to generate the datatypes or not.")
+    val createJsonFormats = settingKey[Boolean]("Whether to generate the JsonFormats for the datatypes or not.")
     val datatypeJavaLazy = settingKey[String]("Interface to use to provide laziness in Java.")
     val datatypeSource = settingKey[File]("Datatype source directory.")
     val datatypeScalaFileNames = settingKey[Definition => File]("Mapping from `Definition` to file for Scala generator.")
     val datatypeScalaSealProtocols = settingKey[Boolean]("Seal abstract classes representing `Protocol`s in Scala.")
+    val datatypeSerializerPackage = settingKey[Option[String]]("Package in which to put the serializer object.")
```

Comment:
I think we should avoid the term "serializer" and call it "Json protocol" or "codec".